src/components/AddNewBook.js

```diff
@@ -1,30 +1,49 @@
-import React, { useState } from 'react';
+import { useState } from 'react';
 import { useDispatch } from 'react-redux';
-import { nanoid } from 'nanoid';
+import { v4 as uuidv4 } from 'uuid';
 import { addBook } from '../redux/books/books';
 
 function AddNewBook() {
-  const [title, setTitle] = useState('');
-  const [author, setAuthor] = useState('');
+  const [cue, setCue] = useState({
+    title: '',
+    author: '',
+  });
   const dispatch = useDispatch();
+
+  const manageInfo = (e) => {
+    const input = e.target.value;
+
+    switch (e.target.id) {
+      case 'title':
+        setCue((inputs) => ({ ...inputs, title: input }));
+        break;
+      case 'author':
+        setCue((inputs) => ({ ...inputs, author: input }));
+        break;
+      default:
+        break;
+    }
+  };
+
   const handleSubmit = (e) => {
     e.preventDefault();
-    const book = {
-      id: nanoid(),
-      title,
-      author,
+    const newBook = {
+      item_id: uuidv4(),
+      title: cue.title,
+      author: cue.author,
+      category: 'none',
     };
-    setTitle('');
-    setAuthor('');
-    dispatch(addBook(book));
+    dispatch(addBook(newBook));
   };
-
   return (
-    <form className="book-form" onSubmit={handleSubmit}>
-      <input className="data-input" type="text" placeholder="Book Title" value={title} onChange={(e) => setTitle(e.target.value)} />
-      <input className="data-input" type="text" placeholder="Book Author" value={author} onChange={(e) => setAuthor(e.target.value)} />
-      <button type="submit">ADD BOOK</button>
-    </form>
+    <div className="add-new-book">
+      <h2>ADD NEW BOOK</h2>
+      <form>
+        <input id="title" onChange={manageInfo} placeholder="Book Title" value={cue.title} />
+        <input id="author" onChange={manageInfo} placeholder="Book Author" value={cue.author} />
+        <button onClick={handleSubmit} type="submit">ADD BOOK</button>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you make all your inputs from required to prevent users to submit an empty form. You can fix by only add the `required` attribute in both of them

- [ **OPTIONAL** ] I think it would be better if you add a feature that cleans your inputs once a new form has added to make your app looks professional. Right now, when a new book has been added, their text remain in the input fields.